crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -422,6 +435,26 @@ impl LogletConfiguration {
             }
         }
     }
+
+    fn node_set_iter(&self) -> impl Iterator<Item = &PlainNodeId> {
+        match self {
+            LogletConfiguration::Replicated(configuration) => {
+                itertools::Either::Left(configuration.nodeset.iter())
+            }
+            LogletConfiguration::Local(_) => itertools::Either::Right(iter::empty()),
+            LogletConfiguration::Memory(_) => itertools::Either::Right(iter::empty()),
+        }
+    }
+
+    fn sequencer_node(&self) -> Option<PlainNodeId> {
+        match self {
+            LogletConfiguration::Replicated(configuration) => {
+                Some(configuration.sequencer.as_plain())
```

Comment:
I'm not entirely sure why is this a plain node id.

Reply:
I think this is wrong and can be a `NodeId` as you've suggested.